.travis.yml

```diff
@@ -11,13 +11,6 @@ php:
 
 matrix:
   fast_finish: true
-  include:
-    - php: 5.5
-      dist: precise
-    - php: 5.4
-      dist: precise
-    - php: 5.3
-      dist: precise
```

Comment:
If we're dropping 5.3 - 5.5 support in this PR for the 1.7.x series, then we should bump the composer.json as well.

https://github.com/bovigo/vfsStream/blob/926040619cb9d084aee75a2da1cd5cd61f33c39a/composer.json#L20

Reply:
You are right, just adjusted that. Anybody still running these PHP versions has larger problems than upgrading to the next version of vfsStream, I guess, so it should be safe to drop support for those.